src/Evolution/Systems/GrMhd/GhValenciaDivClean/Subcell/InitialDataTci.hpp

```diff
@@ -51,9 +51,22 @@ struct DgInitialDataTci {
           gr::Tags::SpacetimeMetric<3, Frame::Inertial, DataVector>,
           GeneralizedHarmonic::Tags::Pi<3, Frame::Inertial>,
           GeneralizedHarmonic::Tags::Phi<3, Frame::Inertial>,
-          ValenciaDivClean::Tags::TildeD, ValenciaDivClean::Tags::TildeTau,
-          ValenciaDivClean::Tags::TildeS<>, ValenciaDivClean::Tags::TildeB<>,
-          ValenciaDivClean::Tags::TildePhi>>& dg_vars,
+          //     const Variables<tmpl::list<
+          //         Inactive<gr::Tags::SpacetimeMetric<3, Frame::Inertial,
+          //         DataVector>>, Inactive<GeneralizedHarmonic::Tags::Pi<3,
+          //         Frame::Inertial>>,
+          //         Inactive<GeneralizedHarmonic::Tags::Phi<3,
+          //         Frame::Inertial>>,
```

Comment:
what's going on here?

Reply:
This was left over from a merge with develop. I removed it.